docs/metrics/extending-the-sdk/README.md

```diff
@@ -28,7 +28,10 @@ not covered by the built-in exporters:
 * Exporters should avoid generating telemetry and causing live-loop, this can be
   done via `OpenTelemetry.SuppressInstrumentationScope`.
 * Exporters receives a batch of `Metric`, and each `Metric`
-  can contain 1 or more `MetricPoint`s.
+  can contain 1 or more `MetricPoint`s. After export, users are expected to
+  perform all actions with the exported Metrics and MetricPoints. Once control
+  is returned, exporter can no longer assume the state of the Batch.
+  (TODO: I need help with the statement here.)
```

Comment:
I need help phrasing this correctly.

Reply:
Should this be "Once control is returned to the exporter, it can no longer assume the state of the Batch."

Reply:
Maybe "the exporter should perform all actions with the Metrics and MetricsPoints in the Batch before returning control from `Export`, once the control is returned, the exporter can no longer assume the state of the Batch."